Surface_mesh/include/CGAL/Surface_mesh/Surface_mesh.h

```diff
@@ -1697,7 +1697,7 @@ class Surface_mesh
     /// With the default value for
     /// `check_all_incident_halfedges` the function iteratates over the incident halfedges.
     /// With `check_all_incident_halfedges == false` the function returns `true`, if the incident
-    /// halfedge associated to vertex `v` is a border halfedge.
+    /// halfedge associated to vertex `v` is a border halfedge, or if the vertex is isolated.
```

Comment:
I changed that and documented it properly, also in the paragraph above the documentation.